.github/workflows/build_test_images.yaml

```diff
@@ -73,6 +80,7 @@ jobs:
             name: ${{ steps.publish-image.outputs.image-name }}
             url: ${{ steps.publish-image.outputs.image-url }}
             checksum: ${{ steps.publish-image.outputs.image-checksum }}
+            cosign-bundle-url: ${{ steps.publish-image.outputs.cosign-bundle-url }}
```

Comment:
So I have modified the manifest. but the purge script probably does need a look.